packages/abi-gen/templates/TypeScript/contract.handlebars

```diff
@@ -392,8 +392,9 @@ export class {{contractName}}Contract extends BaseContract {
         txDefaults?: Partial<TxData>,
         logDecodeDependencies?: { [contractName: string]: ContractAbi },
         deployedBytecode: string | undefined = {{contractName}}Contract.deployedBytecode,
+        jsonRpcIdNameSpace?: string,
     ) {
-        super('{{contractName}}', {{contractName}}Contract.ABI(), address, supportedProvider, txDefaults, logDecodeDependencies, deployedBytecode);
+        super('{{contractName}}', {{contractName}}Contract.ABI(), address, supportedProvider, txDefaults, logDecodeDependencies, deployedBytecode, jsonRpcIdNameSpace);
```

Comment:
can we provide a default value based on the contract name, so that this change doesn't break all existing instantiations?